src/obfl-specification.html

```diff
@@ -1027,7 +1070,11 @@ <h4 id="L906" class="include">sequence</h4>
 		<dt>break-before [optional]</dt>
           <dd class="no-markup">Inserts a break before the sequence begins.</dd>
           <dd>This property only applies to text body sequences.</dd> 
-          <dd>One of 'auto' (default), 'volume'.</dd>
+          <dd>One of 'auto' (default), 'sheet', 'page' or 'volume'.</dd>
+		<dt>break-after [optional]</dt>
+          <dd class="no-markup">Inserts a break after the sequence has ended.</dd>
+          <dd>This property only applies to text body sequences.</dd> 
```

Comment:
> This property only applies to text body sequences.

Why this limitation? Actually I don't think we can do without this attribute on sequences in pre-content.

Reply:
Removed.

Reply:
@bertfrees Although I have removed the "This property only applies to text body sequences." I think it is kind of strange that this limitation is present in the `break-before` attribute. I don't see why the limitation should be present on `break-before` but not on `break-after`.

I don't want to mention that this attribute is not needed in sequences in pre-content. It is also not mentioned that `break-before` is not needed on sequences in post-content.

Reply:
Ah, you mean that sentence was already there for `break-before`? OK I understand. It was  referring to the value "volume" (which used to be the only value), which indeed does not make sense to be used inside `pre-content` and `post-content`. So I guess we should keep the sentence but move it to the description of the "volume" value. I also wonder whether it would be worth having this reflected in the schema?

Reply:
I have been thinking about this one. It is certainly possible to do so but it makes the schema more complicated. My personal opinion is that a schema should be as strict as possible, but it should still be well-structured. I will do a try.

Reply:
Done.